src/core/execute.c

```diff
@@ -456,16 +455,15 @@ int exec_spawn(Unit *unit,
                         &pidref);
         if (r < 0)
                 return log_unit_error_errno(unit, r, "Failed to spawn executor: %m");
-        if (r > 0)
-                move_child = false; /* Already in the right cgroup thanks to CLONE_INTO_CGROUP */
-
-        log_unit_debug(unit, "Forked %s as "PID_FMT, command->path, pidref.pid);
 
```

Comment:
Re-using r in a new block makes it easy to later insert some other call in the middle by mistake, given it's a generic variable. So please remove the newline here, so that it appears as one block.